package.json

```diff
@@ -69,9 +69,14 @@
 		"zh_TW": "繁體中文"
 	},
 	"dependencies": {
+		"@babel/plugin-transform-regenerator": "^7.14.5",
+		"@babel/plugin-transform-runtime": "^7.14.5",
+		"@babel/runtime": "^7.14.6",
```

Comment:
another note - according to https://core.trac.wordpress.org/ticket/52941#comment:24 it appears babel/polyfill will be returning to wordpress rather than the regenerator runtime. Either way this shouldn't be necessary once the change gets into the next nightly build.

Reply:
This also applies to: https://github.com/godaddy-wordpress/coblocks/pull/1923/files#diff-9fd21b09a0144355cfd001e3952767df08b2e04241994470aa1d627f797b171e and https://github.com/godaddy-wordpress/coblocks/pull/1923/files#diff-379120de71991cfea15f15a1225d1ba83a7a50e55a356c831f5f5e0be305ce0b